docs/guides/instances.mdx

```diff
@@ -96,7 +96,9 @@ When determining which jobs to run, the [fair-share scheduler](/guides/fair-shar
 The jobs you run and the jobs run by other collaborators in the same instance count toward the reported usage for that instance. You can see the usage for an instance on your [dashboard](https://quantum.ibm.com/) (use the [instance switcher](#switch-instances) at the top of the page to change which instance is reflected on your dashboard).
 </LegacyContent>
 <CloudContent>
-The jobs you run and the jobs run by other collaborators in the same instance count toward the reported usage for that instance. You can see the usage for an instance on your [dashboard](https://quantum.cloud.ibm.com/).
+The jobs you run and the jobs run by other collaborators in the same instance count toward the reported usage for that instance.
+
+You can view an instance's usage on the [Instances](https://quantum.cloud.ibm.com/instances) page, or, for those with the proper authority, the [Analytics](https://quantum.cloud.ibm.com/analytics) page.  Note that the pages might show different usage numbers because The Instance page shows usage in a 28-day rolling window prior to the current timestamp, while the Analytics page uses full day boundaries.
```

Comment:
```suggestion
You can view an instance's usage on the [Instances](https://quantum.cloud.ibm.com/instances) page, or, for those with the proper authority, the [Analytics](https://quantum.cloud.ibm.com/analytics) page.  Note that the pages might show different usage numbers because the Instance page shows usage in a 28-day rolling window prior to the current timestamp, while the Analytics page uses full-day boundaries.
```